src/store/modules/timelapse.js

```diff
@@ -38,18 +42,50 @@ const getters = {
   times({ startDate, endDate }) {
     //takes two dates and returns an array of ISO date strings
     const daysDifference = differenceInDays(endDate, startDate);
-    const viableDayFractions = [1, 2, 3, 4, 6, 12];
-    for (let dayFraction of viableDayFractions) {
-      //splitting days into numbers of hours
-      for (let j = 12; j < 24; j++) {
-        //splitting timelapse bar itself into fractions
-        if ((daysDifference * dayFraction) % j === 0) {
-          let workingDate = startDate;
-          let timeArray = [];
-          for (let k = 0; k <= j; k++) {
-            //populate array of date strings
-            timeArray[k] = workingDate.toISOString();
-            workingDate = addHours(workingDate, (daysDifference / j) * 24);
+    let timeArray = [];
+    if (daysDifference === 0 || !daysDifference) {
```

Comment:
You might be able to refactor `daysDifference === 0 || !daysDifference` to just `!daysDifference` since JS treats 0 as falsy.